dotnetv3/S3/README.md

```diff
@@ -0,0 +1,105 @@
+# Amazon Simple Storage Service (Amazon S3) code examples for .NET
+
+## Purpose
+
+Customers of all sizes and industries can use Amazon S3 to store and
+protect any amount of data for a range of use cases, such as data lakes,
+websites, mobile applications, backup and restore, archive, enterprise
+applications, IoT devices, and big data analytics. Amazon S3 provides
+management features so that you can optimize, organize, and configure access
+to your data to meet your specific business, organizational, and compliance
+requirements.
+
+The code examples in this directory demonstrate how to work with Amazon S3
+features using the AWS SDK for .NET v3.5 or later.
+
+## Code examples
+
+- [AbortMPUExample](AbortMPUExample/) (`AbortMultipartUploadsAsync`)
+- [BucketACLExample](BucketACLExample/) (`PutBucketACLAsync`, `GetBucketAclAsync`, )
+- [CopyObjectExample](CopyObjectExample/) (`CopyObjectAsync`)
+- [CreateBucketExample](CreateBucketExample/) (`CreateBucketAsync`)
+- [CrossRegionReplicationExample](CrossRegionReplicationExample/)
+(`PutBucketReplicationAsync`, `GetBucketReplicationAsync`)
+- [DeleteBucketExample](DeleteBucketExample/) (`DeleteBucketAsync`)
+- [DualStackEndpointExample](DualStackEndpointExample/) (`ListObjectsV2Async`)
+- [EnableNotificationsExample](EnableNotificationsExample/) (`PutBucketNotificationAsync`)
+- [GenPresignedUrlExample](GenPresignedUrlExample/) (`GetPreSignedURLAsync`)
```

Comment:
This link is broken.

Reply:
Fixed, changed case to match directory name